### ISSUE TITLE
Fail to to debug the python project with an error on Linux

### ISSUE THREAD

**Issue by v-xinda** [microsoft/vscode-docker#3514]
**OS:** Ubuntu20.04
**Docker Extension Build Version:** 20220506.1
**Python Version:** 3.9.12

**Repro Steps:**
1. Open a Python project in VS Code.
2. Add Docker files to it -> Debug.
3. Check whether succeeds to debug the python project.

**Expect:**
Succeed to to debug the python project.

**Actual:**
Fail to to debug the python project with an error.
![image](https://user-images.githubusercontent.com/57662131/167396068-3ffd854e-10f3-4f90-bd3b-b8ed4575b59a.png)

**More Info:**
1. This issue doesn't reproduce on Windows and Mac OS.

**Comment by bwateratmsft** [comment]
I am able to reproduce. Taking a look.

**Comment by bwateratmsft** [comment]
As a workaround to any users who might be hitting this issue, you can add this `"debugAdapterHost": "localhost"` to your `launch.json` file:

```jsonc
{
    "configurations": [
        // ...
        {
            "name": "Docker: Python - Fastapi",
            "type": "docker",
            "request": "launch",
            "preLaunchTask": "docker-run: debug",
            "python": {
                "pathMappings": [
                    {
                        "localRoot": "${workspaceFolder}",
                        "remoteRoot": "/app"
                    }
                ],
                "projectType": "fastapi"
            },
            "debugAdapterHost": "localhost" // Add this line
        }
        // ...
    ]
}
```

VSCode will put yellow warning squiggles in, but you can ignore this. The warning will not show up in the Problems tab once the launch.json file is closed.

**Comment by bwateratmsft** [comment]
This has now been released in Docker extension version 1.23.0.
